### DISCUSSION TITLE
UCI - Error while trying to run locally

### DISCUSSION THREAD

**Discussion by pritamps** [Samagra-Development/SamagraX-community#36] [Installation and Configuration]
I followed the instructions in the [UCI Repo](https://github.com/Samagra-Development/PDF-Package#-installation-with-docker).

The docker containers built successfully, but it seems like `pdf-package_kafka-consumer-1` is causing some trouble.

In the terminal, I see this:

```
pdf-package_kafka-consumer_1 exited with code 0
```

And in the docker desktop, it keeps constantly restarting:

![image](https://user-images.githubusercontent.com/16005325/120480248-75adc800-c3cc-11eb-87eb-7d7b6f022198.png)

How can I resolve this?

@aayushkedia95, @ChakshuGautam, @sukhpreetsamagra

**Comment by aayushkedia95** [comment]
@pritamps, can we get on a call today to discuss this?

cc - @ChakshuGautam
